### ISSUE TITLE
Fix Jest config deprecation warnings

### ISSUE THREAD

**Issue by lassik** [Unibeautify/unibeautify#200]
From the latest Travis log:

```
$ jest --maxWorkers=4

ts-jest[backports] (WARN) "[jest-config].globals.ts-jest.tsConfigFile" is deprecated, use "[jest-config].globals.ts-jest.tsConfig" instead.
ts-jest[backports] (WARN) "[jest-config].globals.ts-jest.enableTsDiagnostics" is deprecated, use "[jest-config].globals.ts-jest.diagnostics" instead.
ts-jest[backports] (WARN) Your Jest configuration is outdated. Use the CLI to help migrating it: ts-jest config:migrate <config-file>.

ts-jest[backports] (WARN) "[jest-config].globals.ts-jest.tsConfigFile" is deprecated, use "[jest-config].globals.ts-jest.tsConfig" instead.
ts-jest[backports] (WARN) "[jest-config].globals.ts-jest.enableTsDiagnostics" is deprecated, use "[jest-config].globals.ts-jest.diagnostics" instead.
ts-jest[backports] (WARN) Your Jest configuration is outdated. Use the CLI to help migrating it: ts-jest config:migrate <config-file>.

ts-jest[backports] (WARN) "[jest-config].globals.ts-jest.tsConfigFile" is deprecated, use "[jest-config].globals.ts-jest.tsConfig" instead.
ts-jest[backports] (WARN) "[jest-config].globals.ts-jest.enableTsDiagnostics" is deprecated, use "[jest-config].globals.ts-jest.diagnostics" instead.
ts-jest[backports] (WARN) Your Jest configuration is outdated. Use the CLI to help migrating it: ts-jest config:migrate <config-file>.

ts-jest[backports] (WARN) "[jest-config].globals.ts-jest.tsConfigFile" is deprecated, use "[jest-config].globals.ts-jest.tsConfig" instead.
ts-jest[backports] (WARN) "[jest-config].globals.ts-jest.enableTsDiagnostics" is deprecated, use "[jest-config].globals.ts-jest.diagnostics" instead.
ts-jest[backports] (WARN) Your Jest configuration is outdated. Use the CLI to help migrating it: ts-jest config:migrate <config-file>.
```

**Comment by stevenzeck** [comment]
Looks like we just have to change `tsConfigFile` to `tsConfig` and `enableTsDiagnostics` to `diagnostics`.

**Comment by lassik** [comment]
I can try that in a PR.

**Comment by lassik** [comment]
Closed by #201
